src/elements/content-sidebar/ContentSidebar.js

```diff
@@ -377,7 +379,7 @@ class ContentSidebar extends React.PureComponent<Props, State> {
         const styleClassName = classNames(
             'be bcs',
             {
-                [`bcs-${((selectedView: any): string)}`]: isOpen,
+                [`bcs-${selectedView || ''}`]: isOpen,
```

Comment:
if `selectedView` can be falsy, would it make sense to change the expression on the right to `isOpen && selectedView`?

Reply:
Someone can fix this on master, I'm only fixing it here because it breaks Flow syntax highlighting for the rest of the file in my editor (VSCode).